h/static/scripts/group-forms/components/CreateGroupForm.tsx

```diff
@@ -99,6 +103,24 @@ function TextField({
 }
 
 export default function CreateGroupForm() {
+  const [name, setName] = useState('');
+  const [description, setDescription] = useState('');
+
+  const createGroup = async () => {
+    let response: any;
```

Comment:
Added: https://github.com/hypothesis/h/pull/8811/commits/3d7916e6b972a0bee9b642927f1fe6c967bbe5ea

Also for error responses: https://github.com/hypothesis/h/pull/8811/commits/ee67ab0b587ddd604d76292f404c6e5c78032986

These do seem to be of limited usefulness since I don't think the frontend can assume that the server's responses will always conform to the expected types so the frontend still has to code defensively, as in:

```javascript
if (response.links && response.links.html) {
  setLocation(response.links.html);
} else {
  setErrorMessage('Invalid API response.');
}
```

(ideally that would also test that `response.links.html` is a valid URL string or handle errors from `setLocation()`).

Or:

```javascript
if (responseJSON && responseJSON.reason) {
  message = responseJSON.reason;
} else {
  message = 'API request failed.';
}
```